config/manifests/bases/service-binding-operator.clusterserviceversion.yaml

```diff
@@ -3,12 +3,14 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     alm-examples: '[]'
-    capabilities: Basic Install
+    capabilities: Seamless Upgrades
```

Comment:
`Seamless Upgrades` is something that we need to actually test before claiming it. Let's wait with that until GA.